proto/p4/p4runtime.proto

```diff
@@ -255,10 +255,10 @@ message MeterEntry {
 // meters is not allowed and will return an error. The insertion/deletion
 // should happen as part of INSERT/DELETE on the associated table-entry.
 // For ReadRequest, the scope is defined as follows:
-// - All meter cells for all meters if meter_id = 0 (default).
-// - All meter cells for given meter_id if table_entry.match is empty.
+// - All meter cells for all tables if table_entry is empty (default).
```

Comment:
For style parity with other APIs here ("read all table entries", "read all indirect meters"), I suggest we require that table_entry is always present (i.e. has_table_entry() == true), and table_id=0 implying "all meter cells for all tables".